app.js

```diff
@@ -89,7 +95,7 @@ function installDropHandler (archive) {
 
         var file = files[i++]
         var stream = fileReader(file)
-        var entry = {name: file.fullPath, mtime: Date.now(), ctime: Date.now()}
+        var entry = {name: prefix + file.fullPath, mtime: Date.now(), ctime: Date.now()}
```

Comment:
i think you can do `path.join(prefix, file.fullPath)` just in case `fullPath` has a `/` in front, it'll resolve properly